app/models/imported_client.rb

```diff
@@ -59,6 +59,14 @@ class ImportedClient < NonHmisClient
     where(where)
   end
 
+  scope :visible_to, ->(user) do
+    if user.can_edit_all_clients? || user.can_manage_imported_clients?
```

Comment:
can_manage_imported_clients was missing previously as it was using NonHmisClien#visible_to